nss.yaml

```diff
@@ -0,0 +1,73 @@
+package:
+  name: nss
+  version: 3.94
+  epoch: 0
+  description: "Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications."
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - mercurial
+      - bash
+      - py3-gyp-next
+      - samurai
+      - perl
+      - zlib-dev
+      - libnspr
+      - libnspr-dev
+      - sqlite-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: nss-package-version
+
+pipeline:
+  # we need to use runs because melange dont have a mercurial checkout pipeline
+  - runs: |
+      hg clone -r NSS_${{vars.nss-package-version}}_RTM https://hg.mozilla.org/projects/nss
```

Comment:
Actually, I realized that this is using a Mercurial ~ref, which isn't safe.  We need to reference deps by checksum.  I'm fixing `nspr` here: https://github.com/wolfi-dev/os/pull/7342

Reply:
Change to use fetch